pkg/astnormalization/input_coercion_for_list.go

```diff
@@ -192,6 +192,10 @@ func (i *inputCoercionForListVisitor) walkJsonObject(inputObjDefTypeRef int, dat
 		defer i.popQuery()
 
 		inputValueDefRef := i.definition.InputObjectTypeDefinitionInputValueDefinitionByName(inputObjDefTypeRef, key)
+		// if the inputValueDefRef is -1 then the input value of this variable key does not exist in the input object type so skip
+		if inputValueDefRef == -1 {
```

Comment:
Using `ast.InvalidRef` constant to check an invalid reference would be more readable.